packages/flutter_tools/lib/src/commands/refresh.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+import '../base/context.dart';
+import '../flx.dart';
+import '../runner/flutter_command.dart';
+
+class RefreshCommand extends FlutterCommand {
+  final String name = 'refresh';
+  final String description = 'Reload the Dart code in a Flutter app (Android only).';
```

Comment:
Make it clear that this involves rebuilding the FLX, i.e. that host local changes take effect, as opposed to just restarting the app with the same code as was used before.